api/index.js

```diff
@@ -215,6 +234,71 @@ app.get('/api/v1/batsignal/council-events', async (req, res) => {
   }
 });
 
+/**
+ * Add EDP data to database
+ * @param {key, values} key and values
+ */
+app.post('/api/v1/edp', validateToken, async (req, res) => {
+  try {
+    const { key, values } = req.body;
+    const client = await getClient();
+    const query = `
+    INSERT INTO edp_metric (
+      key,
+      value
+    )
+    VALUES (
+      $1,
+      $2
+    )
+    ON CONFLICT (key) DO UPDATE
+    SET value = EXCLUDED.value
+    WHERE EXCLUDED.key = $1
+    `;
+    const dbres = await client.query(query, [key, JSON.stringify(values)]);
+    res.status(200).json({
+      status: true,
+      data: dbres,
```

Comment:
We can not return DB response to public. Just status is enough

Reply:
Updated response